.github/workflows/build.yml

```diff
@@ -75,6 +75,26 @@ jobs:
           command: build
           args: --no-default-features --workspace --target thumbv6m-none-eabi
 
+  msrv:
+    name: MSRV
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v3
+
+      - name: Extract MSRV from Cargo.toml
+        run: |
+          MSRV=$(cargo metadata --format-version 1 --no-deps | jq -r '.packages[] | select(.name == "multihash") | .rust_version')
+          echo "MSRV=$MSRV" >> $GITHUB_ENV
+
+      - uses: dtolnay/rust-toolchain@v1
+        with:
+          toolchain: ${{ env.MSRV }}
+
+      - uses: Swatinem/rust-cache@v2
+
+      - run: cargo +"$MSRV" build --all-features --package multihash
```

Comment:
Why no building the whole workspace? It should be good enough to just rely on the MSRV of the `multihash` crate.

Reply:
> Why no building the whole workspace? It should be good enough to just rely on the MSRV of the `multihash` crate.

We can set an MSRV for the whole workspace. I think it is good policy to consider breaking MSRV a breaking change.

The other crates have a lot more dependencies so I think being fine-grained here makes sense. I wouldn't want a breaking change in multihash just because some dependency in `multihash-codetable` decides on a new MSRV.

Reply:
We can adjust the job to test the MSRV of each crate if you want.

Personally, I only care about `multihash` and none of the rest :)

Reply:
> We can adjust the job to test the MSRV of each crate if you want.

Would be nice, but I'm also OK with keeping it to `multihash` only. It won't block merging it.